addons/pos_stripe/models/pos_payment_method.py

```diff
@@ -65,7 +65,7 @@ def stripe_connection_token(self):
 
     def _stripe_calculate_amount(self, amount):
         currency = self.journal_id.currency_id or self.company_id.currency_id
-        return int(amount/currency.rounding)
+        return int(round(amount/currency.rounding))
```

Comment:
```suggestion
        return round(amount/currency.rounding)
```
You can remove the `int` call.